contracts/token/TransferManager.sol

```diff
@@ -183,7 +183,34 @@ contract TransferManager is Pausable {
 
         revert("Transfer not possible");
     }
-
+    
+    /**
+    * 
+    *  Require that the from and to addresses are not frozen.
```

Comment:
Isn't the agent allowed to execute the forced transfer even if addresses are frozen?